youbet/src/Bet.sol

```diff
@@ -44,6 +44,11 @@ contract Bet is Initializable, OwnableUpgradeable, UUPSUpgradeable {
     event ProjectCreated(string projectId, string name);
     event RewardClaimed(address user, uint reward);
 
+    modifier taskExist(string memory _taskId) {
+        require(taskIndices[_taskId] != 0, "Task does not exist.");
+        _;
```

Comment:
what is this line for?

Reply:
Extracting the modifier for code reuse. It's currently used in two places to check whether a task exists.